extensions/awt/runtime/src/main/java/io/quarkus/awt/runtime/graal/AwtFeature.java

```diff
@@ -1,23 +1,11 @@
-package io.quarkus.runtime.graal;
+package io.quarkus.awt.runtime.graal;
 
 import org.graalvm.nativeimage.ImageSingletons;
 import org.graalvm.nativeimage.hosted.Feature;
 import org.graalvm.nativeimage.impl.RuntimeClassInitializationSupport;
 
 import com.oracle.svm.core.annotate.AutomaticFeature;
 
-/**
- * Technically, this should live in extensions/awt,
- * but currently all code that relies on JAXB
- * requires at the very least sun.java2d package to be runtime initialized.
- *
- * Having sun.java2d code initialized at build time caused issues,
- * which is why a substitution was set in place to avoid such code making it to the binary:
- * https://github.com/quarkusio/quarkus/commit/ef87e5567cf3ac462a3f12aad4b5b530d9220223
- *
- * So, as long as JAXB graphics code has not been excluded completely from JAXB,
- * it is safer to define all image related packages to be runtime initialized directly in core.
- */
```

Comment:
Because there's no support for marking a package to be runtime initialized. Last time I checked, there's only support for marking specific classes to be runtime initialized. @Sanne right?